tests/conftest.py

```diff
@@ -48,3 +49,30 @@ def undo():
             monkeypatch.setattr('click.echo', old_echo)
 
     return Result
+
+
+@pytest.fixture
```

Comment:
This should be a session-scoped fixture such that the value is only determined once for all tests, not for each test.